tests/lca/imagebasedupgrade/mgmt/upgrade/tests/e2e-upgrade-test.go

```diff
@@ -325,21 +334,55 @@ var _ = Describe(
 			}
 		})
 
-		It("upgrades the connected cluster", reportxml.ID("71362"), func() {
+		It("upgrades the connected cluster to a newer XY version", reportxml.ID("71362"), func() {
 			By("Check if the target cluster is connected")
 			connected, err := cluster.Connected(APIClient)
+			Expect(err).NotTo(HaveOccurred(), "error checking if the cluster is connected")
 
 			if !connected {
 				Skip("Target cluster is disconnected")
 			}
 
+			By("Check if seed and target have the same XY version")
+			upgradedClusterVersionXY, err := ClusterVersionXY(MGMTConfig.SeedClusterInfo.SeedClusterOCPVersion)
```

Comment:
Can we change it to `seedImageClusterVersionXY`. The current wording makes it seem like the upgrade already happened

Reply:
Done. Thanks.